### PR TITLE
added grunt to travis

### PR DESCRIPTION
**Pull request by Chlodochar**
## Description
Grunt always needs to be run to check and build JS and SASS files to get the webinterface going. Travis needs to do this as well. That is what this PR does.

## Motivation and Context
Make sure Travis goes through the entire setup that other people need to do as well. It might find errors during this setup that can be caught early.

## How Has This Been Tested?
It hasn't, we'll see what Travis does.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -75,6 +75,6 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-html-validation');
 
-  grunt.registerTask('default', ['jshint', 'sass', 'cssmin', 'uglify', 'watch']);
+  grunt.registerTask('default', ['jshint', 'sass', 'cssmin', 'uglify']);
```
Comment (MonkeyLeeT) [inline Gruntfile.js]: Hmm, why remove watch?

Comment (Chlodochar) [inline Gruntfile.js]: @MonkeyLeeT Travis will probably wait indefinitely to build it. Watch should not be default.

**Comment by bmds** [comment]
It would make sense to split it out like this
```js
  grunt.registerTask('build', ['jshint', 'sass', 'cssmin', 'uglify']);
  grunt.registerTask('default', ['build', 'watch']);
```

The Gruntfile is primarily for humans so make the default work the best for humans.
It is a minor thing to then get Travis to run `grunt build`

**Comment by Chlodochar** [comment]
That seems like the best solution. Added.

**Comment by AHAAAAAAA** [comment]
rebasez

File: Gruntfile.js (hunk 1 above)
Comment (invisiblek) [inline Gruntfile.js]: but watch is great for developing locally...

it would make using grunt even more painful to remove it =P

**Comment by Chlodochar** [comment]
Ty!

**Pull request merged**